react/states_and_effects/project_memory_card.md

```diff
@@ -0,0 +1,18 @@
+### Introduction
+
+Here we go again with a new project! Let's make sure you understand the concepts so far. The main goal of this project is to concepts and hooks learned thus far to manage and utilize state while fetching and using data from an external API.
+
+### How The Game Works
+
+Go play around [with this student's solution](https://heldersrvio.github.io/memory-card-game/) to find out for yourself how the Memory game works. Although this example uses cartoon characters, you can use anything you like for your game.
+
+### Assignment
+
+<div class="lesson-content__panel" markdown="1">
+
+1.  Create a new React project with `vite`.
```

Comment:
I'm not sure if we need to add this assignment item. "Create a new React project with XYZ tool" isn't future proof.